api/api-rule-violations-known.list

```diff
@@ -245,6 +245,8 @@ API rule violation: names_match,kubevirt.io/client-go/api/v1,LunTarget,ReadOnly
 API rule violation: names_match,kubevirt.io/client-go/api/v1,NetworkConfiguration,NetworkInterface
 API rule violation: names_match,kubevirt.io/client-go/api/v1,PITTimer,Enabled
 API rule violation: names_match,kubevirt.io/client-go/api/v1,RTCTimer,Enabled
+API rule violation: names_match,kubevirt.io/client-go/api/v1,SysprepSource,UserDataConfigMap
+API rule violation: names_match,kubevirt.io/client-go/api/v1,SysprepSource,UserDataSecretRef
```

Comment:
I think these will go away if you drop "UserData" prefix from Go struct field names.

We copied this prefix from cloud-init code (which also has `NetworkDataSecretRef` etc.) but it doesn't really make sense for Sysprep.
(We = Boris and I pair-programmed on some of this PR)

Reply:
Right :)